assets/stylesheets/index.css

```diff
@@ -0,0 +1,134 @@
+@import "./reset.css";
```

Comment:
- [ ] Optional - rather than importing the reset.css file this way you can also simply link the `reset.css` file in your HTML file just like you did with your `index.css`.